mitodl/micromasters
# Update docker configuration

#### Pre-Flight checklist


#### What are the relevant tickets?
None

#### What's this PR do?
Updates docker-compose configuration

#### How should this be manually tested?
Tests should pass, and the website should load in your local environment. On RC you should also see that nothing has broken.
